app/controllers/feedback_controller.rb

```diff
@@ -1,32 +1,35 @@
 class FeedbackController < ApplicationController
-    skip_before_action :authenticate_user!
-    before_action :no_admins
-    before_action :set_service
+  skip_before_action :authenticate_user!
+  before_action :set_service
+  before_action :redirect_if_admin!
 
-    def index
-        @feedback = @service.feedbacks.new
-    end
+  def index
+    @feedback = @service.feedbacks.new
+  end
 
-    def create
-        @feedback = @service.feedbacks.build(feedback_params)
-        if @feedback.save
-            render "create"
-        else
-            render "index"
-        end
+  def create
+    @feedback = @service.feedbacks.build(feedback_params)
+    if @feedback.save
+      render "create"
+    else
+      render "index"
     end
+  end
 
-    private
+  private
 
-    def set_service
-        @service = Service.find(params[:service_id])
-    end
+  def set_service
+    @service = Service.find(params[:service_id])
+  end
 
-    def feedback_params
-        params.require(:feedback).permit(
-            :body,
-            :topic
-        )
-    end
+  def redirect_if_admin!
```

Comment:
@jhackett1 this is an ugly diff because I fixed some formatting in passing, but really all that's changed is I've added this `redirect_if_admin!` method and switched the `before_action`s around 😄